include/IMeshBuffer.h

```diff
@@ -43,6 +43,7 @@ namespace scene
         EVAI_COUNT
     };
 
+	//! Available components' sizes enumeration
```

Comment:
Best thing about this enum is that it can be casted to an int to give you the actual component count :+1: